static-assets/components/cstudio-browse/browseWebDAV.js

```diff
@@ -222,7 +222,7 @@
         $('.cstudio-wcm-result .results').delegate( ".magnify-icon", "click", function() {
             var path = $(this).attr('data-source');
             var type = $(this).attr('data-type');
-            me.magnify(path, type);
+            CStudioAuthoring.Utils.previewAssetDialog(path, type);
```

Comment:
Will `CStudioAuthoring.Utils.previewAssetDialog` make use of the markup on the FTL to create the dialog? There's no FreeMarker output or unique differences between the different browse FTLs; it looks like duplicated HTML. 

Refactor for the pop up template to be on the JavaScript `previewAssetDialog` function and remove the HTML from the FTLs.